docs/react-native/v2/features/join.mdx

```diff
@@ -111,12 +76,11 @@ enum HMSTrackSettingsInitState {
 }
 ```
 
-#### Here's an example to understand the implementation -
+Let's see how this can be achieved in the following steps.
 
-For joining a room with muted audio/video these values need to be set in the `trackSettings` property while building the `HMSSDK` instance as -
+1. First, for joining a Room with Muted Audio / Video `trackSettings` property should be set while building the `HMSSDK` instance as follows -
 
 ```js
-
 // First, create the Track Settings object
 const trackSettings = getTrackSettings();
```

Comment:
Should we move `getTrackSettings` function call line after function declaration and remove "First" word from comment?